pkg/middlewares/circuitbreaker/circuit_breaker.go

```diff
@@ -31,19 +29,19 @@ func New(ctx context.Context, next http.Handler, confCircuitBreaker dynamic.Circ
 	logger.Debugf("Setting up with expression: %s", expression)
 
 	cbOpts := []cbreaker.CircuitBreakerOption{
-		createCircuitBreakerOptionExpression(expression),
+		createCircuitBreakerOptions(expression),
```

Comment:
hm, I'd deliberately renamed this as it only created the `Expression` option.

I did consider leaving it as `createCircuitBreakerOptions`, but then I think we should pass it the whole `confCircuitBreaker` struct, and have it produce all of the relevant options.